tests/test_utils.py

```diff
@@ -159,22 +159,37 @@
     "bio_things": {"class": "BioThings", "property": "bioThings"},
 }
 
+test_disk_storage = [
+    (2, 4000, 16000), 
+    (1000, 4000, 16000), 
+    (2000000, 1900000, 2000000), 
+    (1073741825, 1073741824, 1181116006.4), 
+]
+
+@pytest.fixture()
+def create_temp_query_file(tmp_path, request):
```

Comment:
This function was created to create temporary files of various size. This utilizes: 
* tmp_path: https://docs.pytest.org/en/6.2.x/tmpdir.html which doesn't require clean up 
* using indirect parameterization: https://docs.pytest.org/en/7.1.x/example/parametrize.html#indirect-parametrization